src/main/java/com/epam/ta/reportportal/dao/LaunchRepositoryCustom.java

```diff
@@ -71,4 +73,6 @@
 	 * @return Map of username -> launches count
 	 */
 	Map<String, Integer> countLaunchesGroupedByOwner(Long projectId, String mode, LocalDateTime from);
+
```

Comment:
java docs for interface methods are needed

Reply:
done